help_to_heat/settings.py

```diff
@@ -75,6 +76,7 @@
     "django.contrib.auth.middleware.AuthenticationMiddleware",
     "django.contrib.messages.middleware.MessageMiddleware",
     "django.middleware.clickjacking.XFrameOptionsMiddleware",
+    "help_to_heat.middleware.RequestLoggingMiddleware",
```

Comment:
Is there logic to how these have been placed?

Reply:
the thinking is that the middlewares are layered so
- we log only if the request makes it all the way to the bottom
- we have the exception handler at the top to catch all exceptions that bubble up

Reply:
Sure, expected it was something like that. Mind adding a comment to that effect?

Reply:
sure